CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 #### Fixes
 
+- fix: Remove unnecessary code in `created` method. ([#438](https://github.com/jazzband/django-auditlog/pull/438))
```

Comment:
I think we don't need to mention this here as it's not adding/changing a feature.
Please remove it.